test/controllers/works_controller_test.rb

```diff
@@ -0,0 +1,35 @@
+require "test_helper"
+
+describe WorksController do
+  describe 'index' do
+    it 'can get the works path' do
+      get works_path
+      must_respond_with :success
+    end
+  end
+
+  describe 'new' do
+    it 'can get the works path' do
+      get new_work_path
+      must_respond_with :success
+    end
+  end
+  describe 'show' do
+    it 'can get the works path' do
+      get works_path
```

Comment:
For the show action you need an id

```suggestion
      get work_path(Work.first.id)
```